### PR TITLE
Sidenav collapse

### PR DESCRIPTION
**Pull request by Bibek-Regmi**

### CODE REVIEW

File: src/components/06-miscellaneous/09-search layout/search layout.njk [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                <fa-icon  class="ng-fa-icon" size="7x">
+                    <svg aria-hidden="true" focusable="false" data-prefix="sds" data-icon="search" class="svg-inline--fa fa-search fa-w-16 fa-7x search" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+                        <path fill="currentColor" d="M459.32,436.69,345.51,322.89A167.24,167.24,0,0,0,384,216c0-92.63-75.36-168-168-168S48,123.37,48,216s75.37,168,168,168a167.24,167.24,0,0,0,106.89-38.49l113.8,113.81a16,16,0,0,0,22.63-22.63ZM80,216c0-75,61-136,136-136s136,61,136,136S291,352,216,352,80,291,80,216Z" ></path>
+                    </svg>
+                </fa-icon>
+            </div>
+            <div  class="sds-card__body margin-2" >
+                <h1 class="sds-card__title" >No matches found</h1>
+                <p>We couldn't find a match for your search criteria.</p>
+                <p  Please try another search or go back to previous results.</p>
```
Comment (davereed) [inline src/components/06-miscellaneous/09-search layout/search layout.njk:142]: Missing a closing > on the P tag

File: src/stylesheets/components/_card.scss [hunk 2]
```diff
@@ -4,6 +4,7 @@
 $sds-card-border-width: 1px;
 $sds-card-border-radius: border-radius('md');
 $inner-border-radius: calc(#{$sds-card-border-radius} - #{$sds-card-border-width});
+$theme-font-sans: 'source-sans-pro';
```
Comment (davereed) [inline src/stylesheets/components/_card.scss:7]: remove, not using this

File: src/stylesheets/components/_sidenav.scss [hunk 3]
```diff
@@ -1,3 +1,5 @@
+$theme-font-sans: 'source-sans-pro';
```
Comment (davereed) [inline src/stylesheets/components/_sidenav.scss:1]: remove, not using this

**Review by davereed — CHANGES_REQUESTED** [review]
Run a lint/format on the HTML files, a lot of extra spaces and oddities that can be cleaned up.

File: src/components/06-miscellaneous/09-search layout/search layout.njk (hunk 1 above)
Comment (Bibek-Regmi) [reply src/components/06-miscellaneous/09-search layout/search layout.njk]: Closed p tag

File: src/stylesheets/components/_card.scss (hunk 2 above)
Comment (Bibek-Regmi) [reply src/stylesheets/components/_card.scss]: Updated

File: src/stylesheets/components/_sidenav.scss (hunk 3 above)
Comment (Bibek-Regmi) [reply src/stylesheets/components/_sidenav.scss]: Updated

**Review by yerramshilpa — APPROVED** [review]

**Pull request merged**